.gitignore

```diff
@@ -1,2 +1,3 @@
 # .gitignore
-node_modules/
\ No newline at end of file
+node_modules/
+test.md
```

Comment:
Nice work creating the `.gitignore` file and adding the necessary files👍 but these files haven't been created🥲 You currently do not have the `node_modueles` folder because you haven't' setup linters for the project🥲 also, you haven't set up the `test.md` file as required😉 kindly take a look at the [requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)and make the necessary change😉

your current directory
<img width="303" alt="filedirectory_notest_nonodemodules" src="https://user-images.githubusercontent.com/64643995/219342556-429bd50f-c7ef-4c52-8267-389d4e5ba913.png">

setup test.md instruction
<img width="391" alt="testgitignore" src="https://user-images.githubusercontent.com/64643995/219342908-377792ce-e50e-474e-8030-54534e91da8a.png">